plugins/woocommerce/src/Internal/Admin/Suggestions/PaymentExtensionSuggestions.php

```diff
@@ -1722,11 +1722,6 @@ public function get_country_extensions( string $country_code, string $context =
 				continue;
 			}
 
-			// Determine if the extension should be included based on the store's state, the provided country and context.
```

Comment:
I would keep this as a means to provide a ready-made place for per-extension logic. I know the `is_extension_allowed` method will end up with just returning `true`, but I think we gain more than we lose. WDYT?

Reply:
Yeah, I 100% agree. The first solution I did was returning `true` like you said but then I removed the whole function in 06416e0bf2e0b0f163170db90b0990955e454e8c. However, I agree with you, let's leave it there for now